src/nunit.analyzers.tests/ClassicModelAssertUsage/IsNotNullAndNotNullClassicModelAssertUsageCodeFixTests.cs

```diff
@@ -0,0 +1,91 @@
+using System.Collections.Immutable;
+using Gu.Roslyn.Asserts;
+using Microsoft.CodeAnalysis.CodeFixes;
+using Microsoft.CodeAnalysis.Diagnostics;
+using NUnit.Analyzers.ClassicModelAssertUsage;
+using NUnit.Analyzers.Constants;
+using NUnit.Framework;
+
+namespace NUnit.Analyzers.Tests.ClassicModelAssertUsage
+{
+    [TestFixture]
+    public sealed class IsNotNullAndNotNullClassicModelAssertUsageCodeFixTests
+    {
+        private static readonly DiagnosticAnalyzer analyzer = new ClassicModelAssertUsageAnalyzer();
+        private static readonly CodeFixProvider fix = new IsNotNullAndNotNullClassicModelAssertUsageCodeFix();
+
+        [Test]
+        public void VerifyGetFixableDiagnosticIds()
+        {
+            var fix = new IsNotNullAndNotNullClassicModelAssertUsageCodeFix();
+            var ids = fix.FixableDiagnosticIds.ToImmutableArray();
+
+            Assert.That(ids.Length, Is.EqualTo(2), nameof(ids.Length));
```

Comment:
you assert it in shorter way:
```csharp
Assert.That(ids, Is.EquivalentTo(new[] {AnalyzerIdentifiers.IsNotNullUsage, AnalyzerIdentifiers.NotNullUsage}));
```

no need to log `nameof(...)` in message - in case of failure NUnit will log that on its own

Reply:
I used `IsTrueAndTrueClassicModelAssertUsageCodeFixTests.cs` as a template, where it is also there. Should this change be made in both places?

Reply:
I agree with @Dreamescaper that the assertion could be expressed more clearly - and as a consequence of this also give a better error message. But as this seems to be a more general problem I think is is fine to keep the code as it is in this PR. I'll create another issue to solve all of the occurrences of this problem (or perhaps make it part of #145)